app/controllers/works_controller.rb

```diff
@@ -532,5 +569,22 @@ def migrated?
 
       params[:submit] == "Migrate"
     end
+
+    # @returns false if an error occured
+    def handle_modification_permissions(uneditiable_message: "Can not edit work: #{@work.id} is not editable by #{current_user.uid}",
```

Comment:
There is a typo on `uneditiable_message` but it's consistent so it works

Reply:
Thanks I'll fix the typo in my next one...